packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -36,7 +36,7 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         uint64 time; // Timestamp used when evaluating the request.
         uint32 rollCount; // The number of rounds that a price request has rolled. Informs if a request can be deleted.
         bytes32 identifier; // Identifier that defines how the voters should resolve the request.
-        mapping(uint256 => VoteInstance) voteInstances; // A map containing all votes for this price in various rounds.
+        mapping(uint32 => VoteInstance) voteInstances; // A map containing all votes for this price in various rounds.
```

Comment:
We can use `uint32` the same type used for [roundIds](https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol#L133)